lobsterpy/structuregraph/__init__.py

```diff
@@ -0,0 +1,154 @@
+# Copyright (c) lobsterpy development team
+# Distributed under the terms of a BSD 3-Clause "New" or "Revised" License
+
+"""
+This package provides the modules for generating graph objects using lobsterpy data
+"""
+
+import os
+from pymatgen.core.structure import Structure
+from pymatgen.io.lobster.lobsterenv import LobsterNeighbors
+from pymatgen.io.lobster.outputs import Charge
+from lobsterpy.cohp.analyze import Analysis
+
+
+class GraphObject:
+    """
+    GraphObject class for bonding data from Lobster
+
+    Attributes:
+        sg: return structure_graph object
+    """
+
+    def __init__(
+        self,
+        path_to_calcdir: str,
+        add_additional_data_sg=True,
```

Comment:
Maybe, we can already add an option to decide that COHP data is used for the determination of the bond strengths and bonds.